ecosystem/typescript/sdk/src/indexer/generated/queries.ts

```diff
@@ -90,30 +90,47 @@ export const TokenActivitiesFieldsFragmentDoc = `
   type
 }
     `;
+export const GetAccountCoinsDataCount = `
+    query getAccountCoinsDataCount($address: String) {
+  current_fungible_asset_balances_aggregate(
+    where: {owner_address: {_eq: $address}}
+  ) {
+    aggregate {
+      count
+    }
+  }
+}
+    `;
 export const GetAccountCoinsData = `
-    query getAccountCoinsData($owner_address: String, $offset: Int, $limit: Int) {
-  current_coin_balances(
-    where: {owner_address: {_eq: $owner_address}}
+    query getAccountCoinsData($where_condition: current_fungible_asset_balances_bool_exp!, $offset: Int, $limit: Int, $order_by: [current_fungible_asset_balances_order_by!]) {
+  current_fungible_asset_balances(
+    where: $where_condition
     offset: $offset
     limit: $limit
+    order_by: $order_by
   ) {
     amount
-    coin_type
-    coin_type_hash
+    asset_type
+    is_frozen
+    is_primary
```

Comment:
Does this query also combine secondary store balances?

Reply:
I just copied the [query and available properties available ](https://cloud.hasura.io/public/graphiql?endpoint=https%3A%2F%2Findexer-testnet.staging.gcp.aptosdev.com%2Fv1%2Fgraphql&query=query+MyQuery+%7B%0A++current_fungible_asset_balances%28%0A++++where%3A+%7Bowner_address%3A+%7B_eq%3A+%220x40b880d668a9d68d7d521bec03b92857363339053c28459fdfd8d3d905ff0379%22%7D%7D%0A++%29+%7B%0A++++amount%0A++++asset_type%0A++++is_frozen%0A++++is_primary%0A++++last_transaction_timestamp%0A++++last_transaction_version%0A++++owner_address%0A++++storage_id%0A++++token_standard%0A++++metadata+%7B%0A++++++token_standard%0A++++++symbol%0A++++++supply_aggregator_table_key_v1%0A++++++supply_aggregator_table_handle_v1%0A++++++project_uri%0A++++++name%0A++++++last_transaction_version%0A++++++last_transaction_timestamp%0A++++++icon_uri%0A++++++decimals%0A++++++creator_address%0A++++++asset_type%0A++++%7D%0A++%7D%0A%7D%0A) in Hasura

@bowenyang007 ?

Reply:
I mean which code runs these queries and what do they pass in for these params?

Reply:
that function uses this query https://github.com/aptos-labs/aptos-core/pull/9747/files#diff-94b0850dd1d4ed6691cb74e3cc1260aec141b30500512e6b62d6dfe32888a3bfR828

Reply:
This would include both primary and secondary fungible stores. Do we really want that right away? cc @gregnazario @lightmark @bowenyang007  as well. We might need to filter out secondary stores (is_primary = true in where) for now until ecosystem can support secondary stores e2e (rest of SDK functions such as transfers, wallets, etc.)